Filters/src/DetectorStepFilter_module.cc

```diff
@@ -29,7 +30,16 @@ namespace mu2e {
   class DetectorStepFilter : public art::EDFilter {
     public:
       static constexpr double maxE_ = 1.0e6; // 1 TeV
-      static constexpr unsigned maxN_ = 1000000; 
+      static constexpr unsigned maxN_ = 1000000;
+
+      struct TimeCutConfig {
+	using Name=fhicl::Name;
+	using Comment=fhicl::Comment;
+
+	fhicl::OptionalAtom<double> maxTime { Name("MaximumTime"), Comment("Maximum time for good step (ns since POT)")};
+	fhicl::OptionalAtom<double> minTime { Name("MinimumTime"), Comment("Minimum time for good step (ns since POT)")};
```

Comment:
Hi Dave,
Thank you for updating the config structure.   I think at the moment the code does not do what we want it to do.
One can set minTime cut to 0 and maxTime cut will be activated at zero as well.
Please change OptionalAtom to simple Atom inside the TimeCutConfig structure, so that if a time cut
is enabled both min and max values have to be explicitly specified.
Andrei

Reply:
Done.